assets/js/components/KeyMetrics/ChipTabGroup/Chip.js

```diff
@@ -36,16 +40,34 @@ export default function Chip( {
 	onClick,
 	selectedCount = 0,
 } ) {
+	let icon = null;
+
+	switch ( slug ) {
+		case KEY_METRICS_CURRENT_SELECTION_GROUP_SLUG:
+			icon = (
+				<CheckMark
+					width={ 12 }
+					height={ 12 }
+					className={ `googlesitekit-chip-tab-group__chip-item-svg googlesitekit-chip-tab-group__chip-item-svg__${ slug }` }
+				/>
+			);
+			break;
+		case KEY_METRICS_SUGGESTED_GROUP_SLUG:
+			icon = (
+				<StarFill
+					className={ `googlesitekit-chip-tab-group__chip-item-svg googlesitekit-chip-tab-group__chip-item-svg__${ slug }` }
+					width={ 12 }
+					height={ 12 }
+				/>
+			);
+			break;
+	}
 	return (
 		<Button
 			className={ classnames( 'googlesitekit-chip-tab-group__chip-item', {
 				'googlesitekit-chip-tab-group__chip-item--active': isActive,
 			} ) }
-			icon={
-				slug === KEY_METRICS_CURRENT_SELECTION_GROUP_SLUG ? (
-					<CheckMark width={ 12 } height={ 12 } />
-				) : null
-			}
+			icon={ icon }
```

Comment:
Then after refactoring, we would call Icon here with the props:
```suggestion
                        icon={ <Icon 
                            width={12} 
                            height={12}
                            className={`googlesitekit-chip-tab-group__chip-item-svg googlesitekit-chip-tab-group__chip-item-svg__${ slug }` }
                        }
```

Reply:
Thank you, @zutigrm , this has been refactored in my recent commits.